GooglePay/src/main/java/com/braintreepayments/api/GooglePayRequest.java

```diff
@@ -243,7 +241,7 @@ public String toJson() {
                         .put("parameters", pm.getValue())
                         .put("tokenizationSpecification", tokenizationSpecifications.get(pm.getKey()));
 
-                if (pm.getKey() == "CARD") {
+                if (pm.getKey().equals("CARD")) {
```

Comment:
```suggestion
                if ("CARD".equals(pm.getKey())) {
```
This might not really be necessary here, but in case `pm.getKey()` can ever be null, it might make sense to reverse this to prevent a null pointer exception.